ppb/camera.py

```diff
@@ -110,6 +111,25 @@ def point_is_visible(self, point: Vector) -> bool:
             and self.bottom <= point.y <= self.top
         )
 
+    def sprite_in_view(self, sprite: Sprite) -> bool:
```

Comment:
`in_frame` requires knowing the mental model of separating things into a camera with a frame and a viewport with pixels.

Since we ditched that language entirely, figured changing the name so it was clear what the purpose was was the right move.